### PR TITLE
Update JS and CSS styling

### PR DESCRIPTION
**Pull request by andy7809**
This PR updates the JS of my portfolio to make it more responsive. It reorganizes content in a series of collapsible buttons instead of hidden divs. It also updates some of my personal content

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,86 +1,123 @@
...
-            <div id="title">
-                <h1>Andrew Wiedenmann</h1>
+<body>
+	<!-- Wrapper for all content -->
+	<div id="content">
+		<!-- Wrapper for header items -->
+		<div id="header">
+			<div id="avatarImg">
+				<img src="/images/avatarAndrew.jpeg" alt="Avatar" id="avatar">
             </div>
```
Comment (saileshsimhadri) [inline portfolio/src/main/webapp/index.html:20]: Is this </div> indented wrong?

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -23,47 +35,57 @@
 #body {
     font-size: 22px;
 }
+.collapsible {
+    background-color: #9C89B8;
```
Comment (saileshsimhadri) [inline portfolio/src/main/webapp/style.css:39]: declarations should be alphabetical

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -1,46 +1,24 @@
...
-    //If the clicked element has no displayKey, return
-    if (clickDisplayKey == null) {
-        return;
-    }
-    const sectionId = event.target.parentNode.parentNode.parentNode.id;
-    let popupDiv = document.getElementById(sectionId.concat("Popup"));
-    popupDiv.innerHTML = popupContentMap.get(clickDisplayKey);
-    popupDiv.style.visibility = "visible";
-}
+for (i = 0; i < coll.length; i++) {
```
Comment (saileshsimhadri) [inline portfolio/src/main/webapp/script.js:11]: This seems a little hard to read for me.

Another way to do this is to give each collapsable a unique id. Then, have an onClick on each one of the buttons that calls an expand function, passing in the corresponding id. Function could be something like this.
```suggestion
function expand(id) {
  var collapsables = document.getElementsByClassName("collapsable");
  for (var i = 0; i < collapsables.length; i++) {
    collapsables[i].style.display = "none";
  }
  document.getElementById(id).style.display = "block";
}
```
This seems a bit easier to read, let me know what you think.

**Comment by andy7809** [comment]
I merged PR #8 here, hoping to piggyback off this PR and get those changes into master. #8 fixes a few stylings in style.css that were messed up by my previous pull requests.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (andy7809) [reply portfolio/src/main/webapp/index.html:22]: done

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (andy7809) [reply portfolio/src/main/webapp/style.css:41]: fixed #8

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (andy7809) [reply portfolio/src/main/webapp/script.js:11]: I agree, I think the inline declaration of a function is difficult to read. I will split it into another function with your suggestions and propose a new code solution.

Comment (andy7809) [reply portfolio/src/main/webapp/script.js:11]: Please consider change I implemented in #9.

`for (i = 0; i < coll.length; i++) {
    // Add onclick functionality to every item in coll
    coll[i].addEventListener("click", respondToClick);
}

function respondToClick(clickEvent){
    //The element that was clicked
    const clickedElement = clickEvent.target;
    //The next element in the HTML, which is the content div containing the content in the collapsible
    let content = clickedElement.nextElementSibling;
    // If showing (display is block), hide. Else, set display to block.
    if (content.style.display === "block") {
        content.style.display = "none";
    } else {
        content.style.display = "block";
    }
}`

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -1,109 +1,150 @@
...
-        <!-- Wrapper for header items -->
-        <div id="header">
-            <div id="avatarImg">
-                <img src="/images/avatarAndrew.jpeg" alt="Avatar" id="avatar">
-            </div>
 
-            <div id="title">
-                <h1>Andrew Wiedenmann</h1>
+<head>
+	<!-- Roboto font link -->
```
Comment (wjiamin) [inline portfolio/src/main/webapp/index.html:5]: Prefer 2 spaces over tabs for indentation

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (wjiamin) [reply portfolio/src/main/webapp/style.css:41]: Note the example here: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (andy7809) [reply portfolio/src/main/webapp/index.html:5]: done

Comment (andy7809) [reply portfolio/src/main/webapp/index.html:4]: done

**Comment by andy7809** [comment]
Last commit should fix the last of styling concerns

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (andy7809) [reply portfolio/src/main/webapp/style.css]: Ah I didn't understand. Re-read the style guide and 
e48451c should fix this!

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (wjiamin) [reply portfolio/src/main/webapp/index.html:4]: Is it github's rendering or are the indents much larger than I expected w/2 spaces?

Comment (andy7809) [reply portfolio/src/main/webapp/index.html:4]: Must be a github thing. I just pulled it down and it renders as two spaces locally.

Comment (andy7809) [reply portfolio/src/main/webapp/index.html:4]: Ok I made a new branch, rebased it with andrewDevelopment, and the html has 2-space tabs. I am 99% sure this is a github issue. If that doesn't sound right to anyone let me know!

Comment (wjiamin) [reply portfolio/src/main/webapp/index.html:4]: SGTM!

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (saileshsimhadri) [reply portfolio/src/main/webapp/script.js:11]: Yeah, this makes sense as well. Thanks!

**Pull request merged**